### PR TITLE
Add colorblind friendly mode

### PR DESCRIPTION
**Pull request by mwbernard**
We had an issue with the minimap images not appearing anymore on the live site after this code was merged, so I reverted that commit and will address the problems here before re merging this code.

@ebrelsford do you have any thoughts on why the live site was failing to access the minimap images here.

### CODE REVIEW

File: vacs-map-app/src/components/CropCards.vue [hunk 1]
```diff
@@ -59,7 +63,10 @@ const navigate = (crop) => {
 }
 
 const getUrl = (crop) => {
-  return new URL(`../assets/img/minimaps/${crop}_${selectedModel.value}.svg`, import.meta.url).href
+  const schemeFolder = colorblindFriendly.value ? 'colorblindFriendly/' : 'default/'
```
Comment (mwbernard) [inline vacs-map-app/src/components/CropCards.vue:66]: this is where we get the dynamic image URL for each minimap

File: vacs-map-app/src/components/CropCards.vue [hunk 2]
```diff
@@ -59,7 +63,10 @@ const navigate = (crop) => {
...
+  const url = `../assets/img/minimaps/${schemeFolder}${crop}_${selectedModel.value}.svg`
+
+  return new URL(url, import.meta.url).href
```
Comment (mwbernard) [inline vacs-map-app/src/components/CropCards.vue:69]: this is where we get the dynamic image URL for each minimap

**Comment by mwbernard** [comment]
@ebrelsford Thanks for the help on this! Seems to be working well for me in the built version - so I've updated the script to output there. 

I also added a couple smaller style adjustments for stuff that had been bothering me, so I think this is looking good.

File: vacs-map-app/src/components/GridOverlay.vue [hunk 3]
```diff
@@ -247,8 +250,8 @@ const getCircleColorDiverging = (extent, center) => {
     // const interpolator = d3.interpolateBrBG;
 
     // const interpolator = d3.interpolatePiYG;
-
-    const interpolator = d3.interpolateHsl(divergingScheme.min, divergingScheme.max)
+    const interpolator = d3.interpolateHsl(divergingScheme.value.min, divergingScheme.value.max)
+    console.log(divergingScheme.value)
```
Comment (ebrelsford) [inline vacs-map-app/src/components/GridOverlay.vue:254]: Please remove

**Review by ebrelsford — APPROVED** [review]
Nice, confirmed this should work in prod.

Could be a follow-up issue but as I note in #127 these SVGs are a bit heavy and may be contributing to mobile issues, how hard would it be to generated PNGs instead?

**Comment by mwbernard** [comment]
@ebrelsford there's likely a better way to just generate these pngs directly with canvas, but this was fairly easy to add to the existing code. 

We're still generating svg files, but then using a library to convert those files to pngs and deleting the svgs once that's done. Does that work for you?

**Review by ebrelsford — APPROVED** [review]
Perfect thanks!

**Pull request merged**